packages/unmock-core/src/backend/index.ts

```diff
@@ -101,7 +104,14 @@ export default class NodeBackend {
   }
 
   public get services(): ServiceStoreType {
-    return this.serviceStore;
+    return (this.serviceStore || NodeBackend.dummyStore).services;
+  }
+
+  public updateServices(newService: IObjectToService) {
+    if (this.serviceStore) {
+      return this.serviceStore.updateOrAdd(newService);
+    }
+    return undefined;
```

Comment:
Is it expected one is "updating services" when no service store is defined?

Reply:
Not sure - probably for a future PR. What should we do in that case?